SYCL/External/README.md

```diff
@@ -1,82 +1,75 @@
-# Table of contents
- * [Overview](#overview)
- * [Execution](#execution)
- * [Directory structure](#directory-structure)
- * [CMake parameters](#cmake-parameters)
- * [LIT parameters accepted by LIT executor](#lit-parameters-accepted-by-lit-executor)
- * [LIT features which can be used to configure test execution](#lit-features-which-can-be-used-to-configure-test-execution)
-
 # Overview
-This directory contains SYCL-related infrastructure for running tests which
-sources are stored outside of this repository. Tests are distributed in
-sub-directories. One directory per application/test repository.
-
-# Execution
-```
-git clone <GIT_REPO> # e.g. https://github.com/intel/llvm-test-suite
-cd llvm-test-suite
-mkdir build
-cd build
-# configuring test execution (selecting compiler, target BE and target device,
-#downloading and building RSBench sources from default location)
-cmake -G Ninja \
-        -DTEST_SUITE_SUBDIRS=SYCL \
-        -DTEST_SUITE_LIT=<PATH_TO_llvm-lit> \
-        -DCHECK_SYCL_ALL="opencl:cpu,acc,gpu,host;level_zero:gpu,host" \
-        -DCMAKE_CXX_COMPILER=<PATH_TO_sycl_compiler> \
-        -DSYCL_EXTERNAL_TESTS="RSBench" \
-        ..
-# Build and run full list of SYCL tests
-ninja check
-# Build all tests dependencies
-ninja
-# Run all SYCL tests
-llvm-lit .
-# Get list of available tests
-llvm-lit . --show-tests
-# Run specific test (e.g. RSBench only)
-llvm-lit SYCL/External/RSBench
-# Run tests with parameters
-llvm-lit --param target_devices=host,gpu --param sycl_be=level_zero \
-        --param dpcpp_compiler=path/to/clang++ --param dump_ir=True SYCL/External/RSBench
-```
 
-Notes:
- - it is assumed that LIT framework, FileCheck and other LIT dependencies are
-available in the same directory with llvm-lit.
+"External" tests directory has necessary infrastructure for running tests where
+sources of tests are expected to be located outside of llvm-test-suite
+repository. Each subdirectory represents external repository for corresponding
+application/test.
 
 # Directory structure
 Every sub-directory (e.g. RSBench) contains the following content:
  * **CMakeLists.txt** - CMake configuration file which is used to obtain
- application binary and data files. There are several variables which are
- used by it (see [Main parameters](#main-parameters) section for details)
+ application binary and data files.
  * **\*.test** - test configuration files, containing command lines, test
-   status lines and verification patterns following [LLVM test infrastructure](https://llvm.org/docs/TestingGuide.html).
+   status lines and verification patterns following
+   [LLVM tests infrastructure](https://llvm.org/docs/TestingGuide.html).
```

Comment:
Why is this required? My understanding is that only **lit.local.cfg** is required and this file configures now external benchmark is tested. As you can see below the file name extension is defined by this file and can be something different (e.g. `*.cpp`).